volatility3/framework/plugins/timeliner.py

```diff
@@ -23,6 +23,7 @@ class TimeLinerType(enum.IntEnum):
     MODIFIED = 2
     ACCESSED = 3
     CHANGED = 4
+    BOOTTIME = 5
```

Comment:
I'm slightly concerned about is just adding a new category of time.  This is presumably based on autopsy output.  Is there a standard for this format, or is it just a first-implementer-decides type thing?

Reply:
I recall a whole bunch of stuff getting shoehorned into different timestamps (I imagine this one would fall under created/born)?  What make boottime special over other system events that might happen?  Why isn't this a more generic "system event" or even just "event"?  Boottime seems like it'll have one use in amongst 1000+ records...

Reply:
Yes, I missed that Timeliner is solely for generating a Sleuthkit body file. I'll update it to match the CREATED category.

Reply:
done